### DISCUSSION TITLE
How can I make color work with with opacity?

### DISCUSSION THREAD

**Discussion by iaminamcom** [tailwindlabs/tailwindcss#3327] [Help]
Hi
I want my custom color to work with opacity classes. Like I want to add custom border/background colors work with opacity. Currently we're doing like like `bg-black bg-opacity-10` to get `background-color: rgba(0,0,0,. 3)` but what if we want to do that with custom color.
FYI I did some try by setting variable property in alpha but never worked out.

**Comment by Nick-Mazuk** [comment]
A few months back I had a [similar issue](https://github.com/tailwindlabs/tailwindcss/discussions/2399), and Adam Wathan pointed me to this example:

https://github.com/adamwathan/tailwind-css-variable-text-opacity-demo

It's essentially defining the colors as a callback that accepts opacity values.

Unfortunately, this [does not seem to be working on Tailwind v2](https://github.com/tailwindlabs/tailwindcss/discussions/2927). There is a pull request to fix it: https://github.com/tailwindlabs/tailwindcss/pull/2919 so I'm still on v1.9 in the meantime. If there is a way to achieve what you're looking for in v2, I'd be interested as well.

**Comment by iaminamcom** [comment]
Ok. Please don't forget to let me know. Thank you ❤️
